packages/beacon-node/src/chain/blocks/types.ts

```diff
@@ -1,16 +1,16 @@
-import {ChainForkConfig} from "@lodestar/config";
+import type {ChainForkConfig} from "@lodestar/config";
 import {MaybeValidExecutionStatus} from "@lodestar/fork-choice";
-import {ForkPostDeneb, ForkSeq} from "@lodestar/params";
+import {type ForkPostDeneb, ForkPostFulu, ForkPreFulu, ForkSeq} from "@lodestar/params";
 import {CachedBeaconStateAllForks, DataAvailabilityStatus, computeEpochAtSlot} from "@lodestar/state-transition";
-import {RootHex, SignedBeaconBlock, Slot, deneb} from "@lodestar/types";
+import type {ColumnIndex, RootHex, SignedBeaconBlock, Slot, deneb, fulu} from "@lodestar/types";
 
 export enum BlockInputType {
   // preData is preDeneb
   preData = "preData",
   // data is out of available window, can be used to sync forward and keep adding to forkchoice
   outOfRangeData = "outOfRangeData",
   availableData = "availableData",
-  dataPromise = "dataPromise",
+  dataPromise = "data_promise",
```

Comment:
why is this changed, seems inconsistent
```suggestion
  dataPromise = "dataPromise",
```

Reply:
updated in #8161